collector/lib/CollectorStatsExporter.cpp

```diff
@@ -241,12 +241,20 @@ void CollectorStatsExporter::run() {
 
         nUserspace += userspace;
 
-        if (counters.userspace) counters.userspace->Set(userspace);
-
-        if (counters.parse_micros_total) counters.parse_micros_total->Set(parse_micros_total);
-        if (counters.process_micros_total) counters.process_micros_total->Set(process_micros_total);
-
-        if (counters.parse_micros_avg) counters.parse_micros_avg->Set(userspace ? parse_micros_total / userspace : 0);
+        if (counters.userspace) {
+          counters.userspace->Set(userspace);
+        }
+
+        if (counters.parse_micros_total) {
+          counters.parse_micros_total->Set(parse_micros_total);
+        }
```

Comment:
Can we add one empty space here? It looks a bit off with it being 1 block with 1 if, then another block with 2 ifs, then a third one with 1 if.

Alternatively, we could get all ifs together with no additional space between them.

Reply:
done